src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -299,6 +299,23 @@ public net.minecraft.world.World field_73004_o #rainingStrength
 public net.minecraft.world.World field_73017_q #thunderingStrength
 public net.minecraft.world.World field_73018_p #prevThunderingStrength
 public net.minecraft.world.biome.Biome$Climate
+public net.minecraft.world.biome.Biome$Climate <init>(Lnet/minecraft/world/biome/Biome$RainType;FLnet/minecraft/world/biome/Biome$TemperatureModifier;F)V
+public net.minecraft.world.biome.Biome$Climate field_242460_b # precipitation
+public net.minecraft.world.biome.Biome$Climate field_242461_c # temperature
+public net.minecraft.world.biome.Biome$Climate field_242462_d # temperatureModifier
+public net.minecraft.world.biome.Biome$Climate field_242463_e # downfall
+public net.minecraft.world.biome.BiomeAmbience field_235205_b_ # fogColor
+public net.minecraft.world.biome.BiomeAmbience field_235206_c_ # waterColor
+public net.minecraft.world.biome.BiomeAmbience field_235207_d_ # waterFogColor
+public net.minecraft.world.biome.BiomeAmbience field_235208_e_ # particle
+public net.minecraft.world.biome.BiomeAmbience field_235209_f_ # ambientSound
```

Comment:
All of these have getters, you need to remove the sided annotations with the SAS file.

Reply:
Had no idea that was a thing. Well, TIL.